src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/service/Et1VettingServiceTest.java

```diff
@@ -31,47 +34,85 @@ void setUp() {
     }
 
     @Test
-    void initialBeforeLinkLabel_Exist_shouldReturnBinaryUrl() {
+    void initialBeforeLinkLabel_ZeroAcas_shouldReturnEt1Only() {
+        var et1BinaryUrl1 = "/documents/et1o0c3e-4efd-8886-0dca-1e3876c3178c/binary";
         List<DocumentTypeItem> documentTypeItemList = new ArrayList<>();
-        documentTypeItemList.add(createDocumentTypeItem(ET1_DOC_TYPE,
-                "http://dm-store:8080/documents/et10dcae-4efd-8886-0dca-1e3876c3178c/binary"));
-        documentTypeItemList.add(createDocumentTypeItem(ACAS_DOC_TYPE,
-                "http://dm-store:8080/documents/acas76ce-4ef8ca1e3-8c60-d3d78808dca1/binary"));
+        documentTypeItemList.add(createDocumentTypeItem(ET1_DOC_TYPE, et1BinaryUrl1));
         caseDetails.getCaseData().setDocumentCollection(documentTypeItemList);
 
-        et1VettingService.initialBeforeYouStart(caseDetails);
-        assertEquals(String.format(BEFORE_LINK_LABEL,
-                        "/documents/et10dcae-4efd-8886-0dca-1e3876c3178c/binary",
-                        "/documents/acas76ce-4ef8ca1e3-8c60-d3d78808dca1/binary"),
+        et1VettingService.initialiseEt1Vetting(caseDetails);
+        assertEquals(String.format(BEFORE_LABEL_TEMPLATE, String.format(BEFORE_LABEL_ET1, et1BinaryUrl1), ""),
                 caseDetails.getCaseData().getEt1VettingBeforeYouStart());
     }
 
     @Test
-    void initialBeforeLinkLabel_NotExist_shouldReturnDefaultUrl() {
+    void initialBeforeLinkLabel_FiveAcas_shouldReturnFiveAcas() {
+        var et1BinaryUrl1 = "/documents/et1o0c3e-4efd-8886-0dca-1e3876c3178c/binary";
+        var acasBinaryUrl1 = "/documents/acas1111-4ef8ca1e3-8c60-d3d78808dca1/binary";
+        var acasBinaryUrl2 = "/documents/acas2222-4ef8ca1e3-8c60-d3d78808dca1/binary";
+        var acasBinaryUrl3 = "/documents/acas3333-4ef8ca1e3-8c60-d3d78808dca1/binary";
+        var acasBinaryUrl4 = "/documents/acas4444-4ef8ca1e3-8c60-d3d78808dca1/binary";
+        var acasBinaryUrl5 = "/documents/acas5555-4ef8ca1e3-8c60-d3d78808dca1/binary";
+
+        List<DocumentTypeItem> documentTypeItemList = new ArrayList<>();
+        documentTypeItemList.add(createDocumentTypeItem(ET1_DOC_TYPE, et1BinaryUrl1));
+        documentTypeItemList.add(createDocumentTypeItem(ACAS_DOC_TYPE, acasBinaryUrl1));
+        documentTypeItemList.add(createDocumentTypeItem(ACAS_DOC_TYPE, acasBinaryUrl2));
+        documentTypeItemList.add(createDocumentTypeItem(ACAS_DOC_TYPE, acasBinaryUrl3));
+        documentTypeItemList.add(createDocumentTypeItem(ACAS_DOC_TYPE, acasBinaryUrl4));
+        documentTypeItemList.add(createDocumentTypeItem(ACAS_DOC_TYPE, acasBinaryUrl5));
+        caseDetails.getCaseData().setDocumentCollection(documentTypeItemList);
```

Comment:
Can we do the above lines in the @BeforeEach method? Same applies to the next method, that should be able to reduce  duplicated code.